packages/material-ui/src/Popover/Popover.test.js

```diff
@@ -659,10 +655,6 @@ describe('<Popover />', () => {
           positioningStyle = instance.getPositioningStyle(element);
         });
 
-        after(() => {
-          instance.getAnchorOffset = stub().returns(anchorOffset);
```

Comment:
Was c&p from following tests that stubbed `instance.getAnchorOffset` differently.